packages/backend-tasks/api-report.md

```diff
@@ -16,6 +16,7 @@ export type HumanDuration = HumanDuration_2;
 // @public
 export interface PluginTaskScheduler {
   createScheduledTaskRunner(schedule: TaskScheduleDefinition): TaskRunner;
+  getTaskIds(): Promise<TaskIds>;
```

Comment:
Maybe there's an argument to be made to return the entire tasks instead, along with their definitions. Of course not the signal or the fn etc, but all of the metadata, and could later be extended to include last time it was run, last exit status, last time taken to run, and whatnot if we so desire

Reply:
True, makes sense